tensorflow/compiler/tf2tensorrt/convert/convert_nodes_test.cc

```diff
@@ -4037,15 +4037,16 @@ TEST_F(OpConverterTest, ConvertSlice) {
   }
 }
 
-TEST_F(OpConverterTest, ConvertConv2D) {
+TEST_P(OpConverterTest1, ConvertConv2D) {
   // Get nodedef for Conv2D layer.
+  DataType dtype = tf_dtype;
```

Comment:
I am thinking to use tf_type and trt_type consistently in the TRT code. Shall we do this to the new code and make change to the existing code when we touch them for other reason?

Reply:
Yes, Let's try to use tf_type consistently.